crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -80,6 +82,13 @@ pub enum ObjectStatus {
     WrappedOrDeleted,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
+pub(crate) struct ObjectRef {
```

Comment:
Could we make use of the ObjectRead struct?

Reply:
I tried but it requires `ObjectRead` and its field `NativeObjectRef` to implement `InputObject`

Reply:
Yeah I think it's fine to define a new type for this because it's an `input`.